frontend/src/pages/MyApplication/MyApplication.js

```diff
@@ -1,8 +1,8 @@
 import { useEffect, useMemo, useState } from "react";
 import { generatePath } from "react-router";
 import { useNavigate } from "react-router-dom";
-import { fetchMyApplicationForms } from "../../api/applicationForms";
-import Container from "../../components/@common/Container/Container";
+import { fetchMyApplicationForms } from "../../api/application-forms";
```

Comment:
r : 
```suggestion
import { fetchMyApplicationForms } from "../../api/applicationForms";
```

소피아가 직접 수정하신 건가요? 아마 #612 번이 머지된 후에 최신 상태로 업데이트를 안하신 것 같네요.

Reply:
최신 develop 가져오면서 conflict 정리를 잘못했던 것 같아요. 수정했습니다. 찾아줘서 고마워요! https://github.com/woowacourse/service-apply/pull/645/commits/d066a0f585cb5227aad1f1f92beb6eba2a746472